### ISSUE TITLE
About the log file of Mix-RoPE training

### ISSUE THREAD

**Issue by 0930mcx** [naver-ai/rope-vit#29]
Dear author, thank you for your excellent work. 
I'm republicating the Mix-RoPE recently. Would you like to open-source the log file of Mix-RoPE training?
Thanks and best wishes!

**Comment by BoyangL1** [comment]
Hi, I also hold the same idea.
Would it be possible to provide the training log files of Mixed-RoPE as well?
It would greatly help the community to better reproduce the results and understand the training dynamics.
Thanks again for your wonderful work!

**Comment by bhheo** [comment]
Dear @0930mcx and @BoyangL1 

Following your recommendation, we've released the training logs in the latest commit.
Because the training code was a bit different from the current version, the logs may not exactly match your reproduction results.
However, they are still helpful in checking the loss and accuracy pattern.

Please feel free to use these logs for your research.
Thank you for your helpful suggestion, and I hope your research goes well.

**Comment by BoyangL1** [comment]
Dear @bhheo 
Thank you so much for sharing the training logs! They are very helpful for our reproduction and analysis. I really appreciate your quick response and support!

**Comment by 0930mcx** [comment]
Dear @bhheo 
Thanks very much for your reply. I'm really grateful to your works and reply!
